### ISSUE TITLE
V8 5.4 breaks the "bindings" module

### ISSUE THREAD

**Issue by targos** [nodejs/citgm#176]
https://ci.nodejs.org/view/Node.js-citgm/job/citgm-smoker/376/

* citgm.serialport-v4.0.1
* citgm.leveldown-v1.4.6
* citgm.bcrypt-v0.8.7
* citgm.ref-v1.3.2
* citgm.time-v0.11.4

They all fail with the following stack trace:
```
path.js:7
     throw new TypeError('Path must be a string. Received ' + inspect(path));
     ^
 TypeError: Path must be a string. Received null
     at assertPath (path.js:7:11)
     at dirname (path.js:1326:5)
     at Function.getRoot (/tmp/6617feee-c0fc-4253-a1c2-190dab812002/bcrypt/node_modules/bindings/bindings.js:146:13)
     at bindings (/tmp/6617feee-c0fc-4253-a1c2-190dab812002/bcrypt/node_modules/bindings/bindings.js:55:32)
```

It looks like an incompatibility of `Error.prepareStackTrace` in https://github.com/TooTallNate/node-bindings/blob/master/bindings.js#L102-L134

**Comment by targos** [comment]
It's because now the first element of the stack trace is the `Error.captureStackTrace` call. I wonder if this is expected...

cc @nodejs/v8

![image](https://cloud.githubusercontent.com/assets/2352663/18179130/63991ffa-7081-11e6-87b3-1d83d88c3fec.png)

**Comment by targos** [comment]
Confirmed on Chrome:

53:
![image](https://cloud.githubusercontent.com/assets/2352663/18179297/1a10bb4e-7082-11e6-8fd7-47e8e05d4cee.png)

Canary:
![image](https://cloud.githubusercontent.com/assets/2352663/18179306/2a45c1a8-7082-11e6-95d8-45fb7291e2ba.png)

**Comment by targos** [comment]
V8 bug: https://bugs.chromium.org/p/v8/issues/detail?id=5342

**Comment by ofrobots** [comment]
Fixed in V8 5.4.500.12.
